### DISCUSSION TITLE
No sandbox found

### DISCUSSION THREAD

**Discussion by mbmueller** [ocamllabs/vscode-ocaml-platform#1255] [Q&A]
![image](https://github.com/ocamllabs/vscode-ocaml-platform/assets/20524060/dcc98761-2c0e-4c48-b9f0-8f0a20575133)
I get this message eventhough opam and ocamllsp as well as acamlc is installed correctly and up-to-date. How can I fix this?

**Comment by smorimoto** [comment]
Could not select the opam switch by clicking the "Pick another sandbox" button?
